docs/how-to-customize-a-contract-tx.md

```diff
@@ -24,6 +24,33 @@ Numbers in [] represent index, starting from 0.
 ### Customize
 You can customize a deployment tx builder by overriding its `buildDeployTransaction` method.
```

Comment:
Document `buildDeployTransaction` as in https://scrypt.io/scrypt-ts/how-to-write-a-contract/built-ins#smartcontract-methods.